Uplift/Views/Headers/HomeScreenHeaderView.swift

```diff
@@ -28,29 +29,105 @@ class HomeScreenHeaderView: UIView {
         welcomeMessage.numberOfLines = 0
         welcomeMessage.text = getGreeting()
         addSubview(welcomeMessage)
-
+        
+        statusButton = StatusButton()
+        statusButton.layer.borderColor = UIColor.gray01.cgColor
+        statusButton.layer.borderWidth = 1
+        statusButton.layer.cornerRadius = 12
+        statusButton.addTarget(self, action: #selector(showCapacityView), for: .touchUpInside)
+        addSubview(statusButton)
+        
         setupLayout()
     }
-
+    
     required init?(coder aDecoder: NSCoder) {
         fatalError("init(coder:) has not been implemented")
     }
-
+    
+    @objc func showCapacityView() {
+        if (statusButton.tag == 0) {
+            statusButton.backgroundColor = .gray00
```

Comment:
I think the design has it as white? imo based on the video having that extra contrast of white feels like it would make it more noticeable.